core/scripts/chaincli/handler/handler.go

```diff
@@ -4,18 +4,41 @@ import (
 	"context"
 	"crypto/ecdsa"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"log"
 	"math/big"
+	"net/url"
+	"os"
+	"time"
 
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/core/types"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/manyminds/api2go/jsonapi"
 
+	"github.com/smartcontractkit/chainlink/core/cmd"
 	link "github.com/smartcontractkit/chainlink/core/gethwrappers/generated/link_token_interface"
+	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/config"
+	"github.com/smartcontractkit/chainlink/core/sessions"
+)
+
+const (
+	defaultChainlinkNodeImage = "smartcontract/chainlink:1.5.1-root"
```

Comment:
why? this is very old

Reply:
I just moved this version to this file :) Updated to the latest one